aksel.nav.no/website/components/logging/useAmplitude.ts

```diff
@@ -1,5 +1,6 @@
 import type { Types } from "@amplitude/analytics-browser";
 import { useEffect } from "react";
+import { useCookies } from "react-cookie";
```

Comment:
Would it make sense to just use "document.cookie"? useCookies is just a wrapper for "universal-cookies" that uses document.cookie and parses it

Reply:
Probably. I just threw this together in a hurry. At first glance, it looked cumbersome to access document.cookie directly, but it's probably not _that_ cumbersome.